packages/core/src/checkout-buttons/strategies/googlepay/googlepay-button-strategy.ts

```diff
@@ -160,13 +185,55 @@ export default class GooglePayButtonStrategy implements CheckoutButtonStrategy {
         throw new InvalidArgumentError();
     }
 
+    private async _createBuyNowCart({ buyNowInitializeOptions }: BuyNowInitializeOptions) {
+        if (
+            typeof buyNowInitializeOptions?.getBuyNowCartRequestBody === 'function' &&
+            this._cartRequestSender
```

Comment:
Is there any case when `this._cartRequestSender` will not be available? Can we remove this check?

Reply:
there are no any cases
I have fixed it
thanks